src/pages/index.js

```diff
@@ -27,6 +27,11 @@ export default function IndexPage({ data }) {
             <TopicCard category={cat} key={cat.name} />
           ))}
         </div>
+        <div className="text-center">
+          <Link to="/organizations" className="text-blue-400">
+            View All Organizations &gt;
```

Comment:
Lets use the chevron key
https://www.compart.com/en/unicode/U+203A

Reply:
done, i added it inside a span because it was smaller compared to the text